stdlib/http/src/test/java/org/ballerinalang/stdlib/services/nativeimpl/request/RequestNativeFunctionSuccessTest.java

```diff
@@ -176,26 +176,6 @@ public void testGetContentLength() {
         Assert.assertEquals(payload.length(), ((BString) returnVals[0]).intValue());
     }
 
-    @Test(description = "Test GetContentLength function within a service. Enable this once this method is back in " +
```

Comment:
Why we removing this test?

Reply:
Because getContentLength() method is no longer available in the HTTP request